history-of-the-quakers/original/28-book-3-chapter-2.adoc

```diff
@@ -525,12 +536,14 @@ and during this reign, rigorous in themselves,
 and made more so by the extreme severity exercised in the execution of them,
 were these which follow.
 
-First, The act passed in the thirty-second year of Henry VIII.
```

Comment:
So this was a really tricky area....there are lots of numbered paragraphs, which some seem to be numbered under a number and continued on. There are also indented portions that I can't figure out why they're indented because they aren't attributed to anyone else and they sound like the narrator, and they aren't quoting any parts of the acts or anything. 🤷‍♀

Reply:
yeah, i have no idea what to do with this section either. i think your attempt is as good as anything I could come up with.  Let's go with it for now....